client/src/components/EditMovieForm.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, useHistory } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+
+import axios from 'axios';
+
+const EditMovieForm = (props) => {
+	const { push } = useHistory();
+
+	const [movie, setMovie] = useState({
+		title:"",
+		director: "",
+		genre: "",
+		metascore: 0,
+		description: ""
+	});
+
+    const { id } = useParams();
```

Comment:
Do we want to leave this here? Or maybe just write a comment like:

"// Grab the move id from the URL. Use that id to fetch the movie we are trying to edit"

I'm fine either way. Just a suggestion